en/docs/concepts/user-management/realm.md

```diff
@@ -0,0 +1,51 @@
+# Introduction to Realms
+
+The complete functionality and contents of the User Management module is called a **user realm**. The realm includes the user management
+classes, configurations and repositories that store information. Configuring the User Management functionality in WSO2 Identity Server involves setting up the relevant repositories and updating the relevant configuration files.
+
+---
+
+## Configuring the authorization manager
+
+According to the default configuration in WSO2 Identity Server, the users, roles and permissions are stored in the same repository (i.e., the default, embedded H2 database). However, this configuration can be changed in such a way that the users and roles are stored in one repository (userstore) and the permissions are stored in a separate
+repository. A userstore can be a typical RDBMS, an LDAP or an external Active Directory. The different types of userstores will be discussed in the [userstores](../userstores) section. 
```

Comment:
"configuration can be changed in such a way that the users and roles are stored in one repository (userstore) and the permissions are stored in a separate repository" 

shall we explain why doing this might be useful?

Reply:
an LDAP{comma}, or an...

Reply:
The different types of userstores will be discussed in the [userstores](../userstores) section. --> See [userstores] for more information about the different types of userstores?